### ISSUE TITLE
Battlerager subclass not available for Duergar race

### ISSUE THREAD

**Issue by guilofrano** [AuroraLegacy/elements#220]
The barbarian subclass "Battlerager" has a prerequisite where you need to be a Dwarf in order to get it, but if you choose the Duergar race you don't see it, even though duergars are "...considered a dwarf for any prerequisite or effect that requires you to be a dwarf".

They I fixed it locally was by simply adding "ID_INTERNAL_RACIAL_TRAIT_TYPE_HUMANOID_DWARF" to the battlerager requirements tag, but I don't know if that's the best approach.

**Comment by NickVendel** [comment]
Yes, that is the best approach when it comes to this. I'm pretty sure Battlerager only has requirement of the Dwarf race, so in this case we forgot to add that internal element into its requirements.
